diffusion/models/autoencoder.py

```diff
@@ -586,7 +657,7 @@ def get_metrics(self, is_train: bool = False):
     def update_metric(self, batch, outputs, metric):
         clamped_imgs = outputs['x_recon'].clamp(-1, 1)
         if isinstance(metric, MeanMetric):
-            metric.update(torch.square(outputs['latents']))
+            metric.update(torch.sqrt(torch.square(outputs['latents'])))
```

Comment:
Somewhat of an arbitrary choice, but the quantity with the square root is closer to the value needed for normalizing latents later on. It still isn't quite right though because this gives the mrs instead of the rms. This logging isn't useful if one runs with the `log_latent_statistics` callback, so I could remove it if you think it is misleading.

Reply:
Sorry if I'm missing something, but couldn't this be an `torch.abs` instead of `torch.sqrt(torch.square())`. It doesn't change the code functionally, but the current implementation seems unnecessary to me.

Reply:
Ah, I see what you mean now! Yes, it could just be a `torch.abs`. But, given how the mean metric updates stats, I think it is actually better to leave this as a `torch.square` without the `torch.sqrt`.